File: python/date_time.py
# date_time.py
# This program gets the current date and time and prints them to the console.
# Github: https://www.github.com/0x4248/examples
# Licence: GNU General Public License v3.0
# By: 0x4248

import datetime

now = datetime.datetime.now()  # Get the current date and time

print("Date:", now.strftime("%d-%m-%Y"))  # Print the date in the format DD-MM-YYYY
print("Time:", now.strftime("%H:%M:%S"))  # Print the time in the format HH:MM:SS

File: c/date_time.c
/**
 * @file date_time.c
 * @brief Program to get the current date and time
 * @details This program gets the current date and time and prints them to the
 * console.
 * @github: https://www.github.com/awesomelewis2007/examples
 */

#include <stdio.h>
#include <time.h>

int main() {
    time_t t = time(NULL);
    struct tm *tm = localtime(&t);
    char date[100];
    char time[100];

    strftime(date, sizeof(date), "%d-%m-%Y", tm);
    strftime(time, sizeof(time), "%H:%M:%S", tm);

    printf("Date: %s\n", date);
    printf("Time: %s\n", time);

    return 0;
}
